openstf_dbc/services/prediction_job.py

```diff
@@ -81,23 +115,18 @@ def get_prediction_jobs(
             limit=limit,
         )
 
-        results = _DataInterface.get_instance().exec_sql_query(query)
-
-        if len(results) == 0:
-            return []
-
-        # Convert to list of dictionaries
-        prediction_jobs = results.to_dict(orient="records")
-
-        # Add description to all prediction jobs
-        prediction_jobs = self._add_description_to_prediction_jobs(prediction_jobs)
+        # Retrieve prediction jobs from database
+        prediction_jobs = self._get_prediction_jobs_query_results(query)
 
         # Add quantiles
         prediction_jobs = self._add_quantiles_to_prediciton_jobs(prediction_jobs)
 
         # Add model group
         prediction_jobs = self._add_model_type_group_to_prediction_jobs(prediction_jobs)
 
+        # Change prediction jobs to dataclass
+        prediction_jobs = self._create_prediction_jobs_objects(prediction_jobs)
```

Comment:
Perhaps a separate function is a bit much.
Would this also work:
`prediction_jobs = [self._create_prediction_job_object(prediction_job) for prediction_job in prediction_jobs]`